src/wlserver.cpp

```diff
@@ -1217,7 +1217,7 @@ void wlserver_run(void)
 		.events = POLLIN,
 	};
 	while ( g_bRun ) {
-		int ret = poll( &pollfd, 1, -1 );
+		int ret = poll( &pollfd, 1, 3 );
```

Comment:
I don't think this is a good fix.

Reply:
It's not a fix but a work around until the real problem is fixed. However I don't think it's a good idea to have no timeout as, just like the linked problem show, it can cause problems.